LDMP/calculate_ldn.py

```diff
@@ -318,7 +318,7 @@ def btn_calculate(self):
                 .80,
                 'legend_nesting':
                 LCLegendNesting.Schema().dump(
-                    self.lc_setup_widget.aggregation_dialog.nesting
+                    self.lc_setup_widget.aggregation_dialog._nesting
```

Comment:
Should now be referencing `lc_setup.get_lc_nesting()`?

Reply:
Same...should be the `.nesting` without the underscore